src/elm/Route.elm

```diff
@@ -26,6 +25,7 @@ type Route
     | NewCommunity
     | EditCommunity Symbol
     | NewAction Symbol String
+    | VerifyClaim String String String String
```

Comment:
this should be `VerifyClaim Symbol String String String`

Reply:
Updated to use `Symbol` instead of `String`